ext/date/tests/rfc-datetime_rfc3339_extended.phpt

```diff
@@ -0,0 +1,17 @@
+--TEST--
+RFC: DateTime RFC3339 Extended
```

Comment:
Left over. Ill fix it
On Feb 20, 2015 11:08 AM, "Derick Rethans" <notifications@github.com> wrote:

> In ext/date/tests/rfc-datetime_rfc3339_extended.phpt
> <https://github.com/php/php-src/pull/1103#discussion_r25070967>:
>
> > @@ -0,0 +1,17 @@
> > +--TEST--
> > +RFC: DateTime RFC3339 Extended
>
> This should have the PHP bug ID. Not sure why it says RFC?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/php/php-src/pull/1103/files#r25070967>.
>